WMDE_FR_2025_Desktop_DE_01

Sperling-0 opened the pull request:
- Update documentation to use new banner names
- Implement banner

Ticket: https://phabricator.wikimedia.org/T397692

Sperling-0 committed:
Update documentation to use new banner names

Sperling-0 committed:
Prepare campaign for WMDE_FR_2025_Desktop_DE_01

Ticket: https://phabricator.wikimedia.org/T397692

Sperling-0 committed:
Implement CTRL for WMDE_FR_2025_Desktop_DE_01

- The banner is based on the control banner of desktop-de-23 from last year.
- The average donation amount is changed to "22,89 €".
- The progress bars are removed.
- Clicking the close button doesn't show the soft close, but logs a close event and prevents banner displays for 7 days.
- The banner don't display the campaign day sentence.
- The banner display the general donors vs visitors sentence.
- The banners use the changed use of funds overlay.
- The dark mode color schema is added to the banners.

Ticket: https://phabricator.wikimedia.org/T397692

Sperling-0 committed:
Implement VAR for WMDE_FR_2025_Desktop_DE_01

- The banner form is changed.

Ticket: https://phabricator.wikimedia.org/T397692

Co-authored-by: Gabriel Birke <gabriel.birke@wikimedia.de>

Abban commented on banners/desktop/WMDE_FR_2025_Desktop_DE_01/styles/Banner.scss:
```diff
@@ -0,0 +1,22 @@
+		font-size: 14px;
+		font-family: fonts.$ui;
+		box-shadow: 0 3px 0.6em rgb( 60 60 60 / 40% );
+		background-color: var( --main-background );
```
This background colour should actually be on the `.wmde-banner-message` element, and is wrong on the English banner too. You can see in the screenshot that the background is extending beyond the red-bordered box.

You can use the same css variable but apply it in `src/themes/Treedip/Message/Message.scss`

![Uploading Screenshot 2025-07-01 at 12.28.55.png…]()

Sperling-0 replied:
I applied the changes you suggested. I have still kept the background color in `Banner.scss`, because without that the banner looked like following:
![Screenshot from 2025-07-01 15-03-43](https://github.com/user-attachments/assets/601078a2-dd4e-45a9-bc29-7594a415c7d0)

Abban commented on banners/desktop/WMDE_FR_2025_Desktop_DE_01/styles/MainBanner.scss:
```diff
@@ -0,0 +1,59 @@
+		flex: 1 1 auto;
+		overflow-y: hidden;
+		margin-right: 30px;
+		padding: 0 0 10px;
```
You should remove this padding

Abban commented on banners/desktop/WMDE_FR_2025_Desktop_DE_01/styles/MainBanner.scss:
```diff
@@ -0,0 +1,59 @@
+
+		border: 5px solid var( --message-border );
+		border-radius: 9px;
+		margin-bottom: 10px;
```
You should remove this margin bottom.

Abban reviewed (approved):
Mostly good, I just spotted a bug and showed you where you can remove margin/padding from the message box.

Sperling-0 committed:
Fix background color and spacing

Ticket: https://phabricator.wikimedia.org/T397692

Sperling-0 committed:
Fix slider arrows' padding

- Update the TOML file to have the latest campaign start dates

Ticket: https://phabricator.wikimedia.org/T397692

Abban commented on banners/desktop/WMDE_FR_2025_Desktop_DE_01/components/MainBanner.vue:
```diff
@@ -0,0 +1,46 @@
+tart<template>
```
Spotted this is back

Sperling-0 replied:
I removed it. I hope that is what you meant.
It was a copy-paste error from the previous banner.

Abban commented on banners/desktop/WMDE_FR_2025_Desktop_DE_01/styles/components/SelectGroupImageLabel.scss:
```diff
@@ -0,0 +1,45 @@
+@use '@src/themes/Treedip/variables/breakpoints';
```
```suggestion
@use 'src/themes/Treedip/variables/breakpoints';
```

Sperling-0 replied:
Thanks.
I thought it was IDE being confused.
